sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Resources.resx

```diff
@@ -237,4 +237,28 @@
   <data name="CannotCreateCheckpointForEmptyEvent" xml:space="preserve">
     <value>A checkpoint cannot be created or updated using an empty event.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="HandlerHasAlreadyBeenAssigned" xml:space="preserve">
+    <value>Another handler has already been assigned to this event.</value>
```

Comment:
... and there can be only one!  

Most .NET events support multicast and allow multiple handlers.  We probably want to be specific in calling out that we allow only a single handler.